horace_core/lattice_functions/refine_crystal.m

```diff
@@ -110,21 +111,25 @@
 
 % Perform calculations
 % --------------------
-[b0,arlu,angrlu] = bmatrix(lattice0(1:3),lattice0(4:6));
-[binit,arlu,angrlu] = bmatrix(lattice_init(1:3),lattice_init(4:6));
+b0    = bmatrix(lattice0(1:3),lattice0(4:6));
+binit = bmatrix(lattice_init(1:3),lattice_init(4:6));
 
 vcryst0=b0*rlu0';       % crystal Cartesian coords in reference lattice
 vcryst_init=binit*rlu'; % crystal Cartesian coords in initial lattice
 
 % Check lengths are all non-zero
 lensqr0=sum(vcryst0.^2,1);
 lensqr_init=sum(vcryst_init.^2,1);
-if any(lensqr0<small)||any(lensqr_init<small), error('Check none of the reciprocal lattice vectors are at the origin'), end
+if any(lensqr0<small)||any(lensqr_init<small)
+    error('HORACE:lattice_functions:invalid_argument', ...
+        'Check none of the reciprocal lattice vectors are at the origin')
```

Comment:
Do you mean have 0 length here?

Reply:
no, as I understand, Bragg position [0,0,0] is not allowed. (That's Toby writing)